### PR TITLE
Add Project Details Revuiew Page

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

_Provide a description of what this PR does. Link to any relevant GitHub issues, Notion tasks or Discord discussions._

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

File: src/components/Create/components/pages/ReviewDeploy/components/DescriptionCol/DescriptionCol.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  desc: ReactNode
+  optional?: ReactNode
+  span?: number | string
+  flex?: number | 'none' | 'auto' | string
+}) => {
+  return (
+    <Col span={span} flex={flex}>
+      <div style={{ display: 'flex', flexDirection: 'column', gap: '0.5rem' }}>
+        <div style={headerTextStyle}>{title}</div>
+        <div>{desc ? desc : <i>{optional}</i>}</div>
```
Comment (tomquirk) [inline src/components/Create/components/pages/ReviewDeploy/components/DescriptionCol/DescriptionCol.tsx:22]:
```suggestion
        <div>{desc ?? <i>{optional}</i>}</div>
```

File: src/components/Create/components/pages/ReviewDeploy/components/DescriptionCol/DescriptionCol.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { Col } from 'antd'
+import { ReactNode } from 'react'
+import { headerTextStyle } from '../styles'
+
+export const DescriptionCol = ({
+  title,
+  desc,
+  optional = '-',
```
Comment (tomquirk) [inline src/components/Create/components/pages/ReviewDeploy/components/DescriptionCol/DescriptionCol.tsx:8]: **question** what's this do? Wasn't imemdiately clear to me from the name

File: src/components/Create/components/pages/ReviewDeploy/components/ProjectDetailsReview/ProjectDetailsReview.tsx [hunk 3]
```diff
@@ -1,3 +1,127 @@
...
+          <div
+            style={{ display: 'flex', flexDirection: 'column', gap: '2.5rem' }}
+          >
+            <Row>
+              <DescriptionCol
+                span={8}
+                title={t`Twitter`}
+                desc={
+                  twitter ? (
+                    <div style={emphasisedTextStyle('0.875rem')}>{twitter}</div>
```
Comment (tomquirk) [inline src/components/Create/components/pages/ReviewDeploy/components/ProjectDetailsReview/ProjectDetailsReview.tsx:79]: **question** is it worth having a component for this that we can re-use?

File: src/components/Create/components/pages/ReviewDeploy/components/ProjectDetailsReview/ProjectDetailsReview.tsx [hunk 4]
```diff
@@ -1,3 +1,127 @@
...
+                      <div style={emphasisedTextStyle()}>Pay Now</div>
+                    ) : null
+                  }
+                />
+              </Col>
+              <Col span={16}>
+                <DescriptionCol
+                  flex={1}
+                  title={t`Pay disclaimer`}
+                  desc={payDisclosure ? payDisclosure : null}
```
Comment (tomquirk) [inline src/components/Create/components/pages/ReviewDeploy/components/ProjectDetailsReview/ProjectDetailsReview.tsx:118]:
```suggestion
                  desc={payDisclosure ?? null}
```

**Review by tomquirk — APPROVED** [review]

**Comment by wraeth-eth** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2201?utm_source=stack-comment).


Job Reference: kRykM3jQ6n0MkVnYZETw

**Comment by wraeth-eth** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2201?utm_source=stack-comment).


Job Reference: arvMYClIbaPLJj0BeNOg

**Pull request merged**